test/wasm-contracts/src/processor.ts

```diff
@@ -0,0 +1,40 @@
+import {SubstrateProcessor} from "@subsquid/substrate-processor"
+import {TypeormDatabase} from "@subsquid/typeorm-store"
+import {assertNotNull} from "@subsquid/util-internal"
+import {Flip} from "./model"
+import {Abi} from "@polkadot/api-contract"
+import flipperMetadata from "./metadata.json"
+import assert from "assert"
+
+
+const processor = new SubstrateProcessor(new TypeormDatabase('wasm_contracts'))
+
+
+processor.setDataSource({
+    chain: assertNotNull(process.env.CHAIN_NODE),
+    archive: assertNotNull(process.env.ARCHIVE)
+})
+
+
+processor.addContractsEventHandler(
```

Comment:
as mentioned above, to my ear `processor.addWasmEventHandler` sounds more natural and self-explanatory

Reply:
i don't mind:)
when i chose addContractsEventHandler i was thinking about another pallets which support wasm
gear-tech for example
probably they will have different handler signature